client/controllers/incidentReports/incidentForm.coffee

```diff
@@ -24,6 +25,7 @@ Template.incidentForm.onCreated ->
 
   incident = @data.incident
   if incident
+    @suggestedFields = incident.suggestedFields
```

Comment:
Should we move this up to line 19 like we do [here](https://github.com/ecohealthalliance/eidr-connect/blob/dfb5101952d7b1033fa873607f9e6b51d768b488/client/controllers/incidentReports/suggestedIndidentModal.coffee#L7)?